fzf-tab.zsh

```diff
@@ -365,8 +372,17 @@ toggle-fzf-tab() {
 	  fi
 }
 
+toggle-sort-fzf-tab() {
+	  if [[ ${FZF_TAB_SORT} == 1 ]]; then
+	        FZF_TAB_SORT=0
+	  else
+		    FZF_TAB_SORT=1
+	  fi
+}
+
 enable-fzf-tab
 zle -N toggle-fzf-tab
+zle -N toggle-sort-fzf-tab
```

Comment:
It would be really nice if all public names started with fzf or FZF.

Is it too late to rename?